_includes/site-navigation/sidenav-mobile.html

```diff
@@ -1,126 +1,133 @@
 {% assign current = page.url | downcase | split: '/' %}
 
 <aside class="sidenav-mobile">
-  <a class="sliding-panel-close" href="#">
-    <img src="/assets/img/close.svg" alt="close">
-  </a>
+  <a class="sliding-panel-close usa-button-dismiss" href="#"></a>
```

Comment:
@diego-ruiz-rei - Thanks! Sorry, I should have been more specific. To meet 1194.22(L) we need to offer a text-based alternative as well; that's where the `span` comes into play: `<span class="usa-sr-only">Close</span>`

The `usa-sr-only` class moves the text off-screen, but maintains it as a point for screen readers - hence the `sr-only`. Hope there clarifies.

Thanks again!